index.js

```diff
@@ -612,8 +612,7 @@ module.exports = class Hypercore extends EventEmitter {
     if (this.opened === false) await this.opening
     if (this.closing !== null) return false
 
-    // TODO: add an option where a writer can bootstrap it's state from the network also
-    if (this.writable) {
+    if (this.writable && !opts?.force === true) {
```

Comment:
Can you make this without the ?. syntax? A bit too new for Hypercore.
Otherwise, looks great and love the test :)

Reply:
Thanks. Done.